source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,151 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+convenient methods for different types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <geospatial-indexes>`
+
+In a textbook, the index provides the user a way to view pages a
+keyword they're looking for is. Likewise, an index in MongoDB groups
+data based on the field you provide and by an index builder mentioned
+above. See the :doc:`Indexes </fundamentals/indexes>` page to see
+examples of queries on indexes. 
+
+The :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+class provides static factory methods for all the MongoDB index key
+types. Each method returns a :ref:`Bson <bson>` instance, which you can
+pass to :java-sync-api:`createIndex() <com/mongodb/client/MongoCollection.html#createIndex(com.mongodb.client.ClientSession,org.bson.conversions.Bson)>`.
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Indexes.*;
+
+   The examples below assume this static import.
+
+.. _index-ascending:
+
+Ascending
```

Comment:
You mean the order of ascending and descending don't matter in a compound index?

Reply:
No, maybe I need to be more clear in my comment-writing. You can read it as follows:

Somewhere, we should mention that if we only specify a single index key -- not a compound one -- it does not matter whether we create an ascending or descending index because MongoDB can scan both directions.